### ISSUE TITLE
Links in generated example code are outdated

### ISSUE THREAD

**Issue by frankbenoit** [eclipse/xtext#1556]
Hi

this generated comment:
````
/**
 * Provides labels for EObjects.
 * 
 * See https://www.eclipse.org/Xtext/documentation/304_ide_concepts.html#label-provider
 */
````
So this link:
  https://www.eclipse.org/Xtext/documentation/304_ide_concepts.html#label-provider
should be:
  https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html

There are more such links in the generated code.

regards Frank

**Comment by miklossy** [comment]
Thanks for the bug report. I will check all the links and replace them with the correct one.

**Comment by cdietrich** [comment]
@miklossy are you still working on this issue

**Comment by miklossy** [comment]
Yes, I will provide a PR asap

**Comment by miklossy** [comment]
Fixed in 2.20
